<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>289</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="numBox1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="numBox2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="numBox3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="numBox4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>â†’</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="targetBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="resultList"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>numBox1</tabstop>
  <tabstop>numBox2</tabstop>
  <tabstop>numBox3</tabstop>
  <tabstop>numBox4</tabstop>
  <tabstop>targetBox</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
